assets/js/index.js

```diff
@@ -88,5 +88,6 @@
     });
 
     setupSelect2For('location');
+    $('select[name="source"]').select2();
```

Comment:
I'm not sure (and haven't checked the API docs), but for these sources we don't want to allow the user to search for a source name that doesn't exist in our data. This is different from the `location` select, where the user can add anything. Is this the case here?